pymc3/variational/advi.py

```diff
@@ -163,7 +163,7 @@ def advi(vars=None, start=None, model=None, n=5000, accurate_elbo=False,
                 avg_elbo = elbos[i - n // 10:i].mean()
                 progress.set_description('Average ELBO = {:,.5g}'.format(avg_elbo))
 
-            if i % eval_elbo == 0:
+            if i > 0 and i % eval_elbo == 0:
                 elbo_prev = elbo_current
                 elbo_current = elbos[i]
                 delta_elbo = abs((elbo_current - elbo_prev) / elbo_prev)
```

Comment:
@lbollar Why is `circ_buff.append` inside the `i % eval_elbo == 0` branch? Shouldn't this be executed at every iteration?